install_config/install/advanced_install.adoc

```diff
@@ -335,6 +335,30 @@ default is
 If you alter this variable, ensure the host name is accessible via your router.
 |===
 
+[[advanced-install-configuring-registry-location]]
+=== Configuring a Registry Location
+
+If you are using a registry other than the default `registry.access.redhat.com`,
+specify the desired registry within the *_/etc/ansible/hosts_* file.
+
+----
+oreg_url=example.com/openshift3/ose-${component}:${version}
+openshift_examples_modify_imagestreams=true
+----
+
+.Registry Variables
+[options="header"]
+|===
+
+|Variable |Purpose
+|`*oreg_url*`
+|Set the alternate image location. Necessary if you are not using the default registry at `registry.access.redhat.com`.
+
+|`*openshift_examples_modify_imagestreams*`
+|Set to `true` if pointing to a registry other than the default.
```

Comment:
I am wondering that it should mention that following two things:
1. `openshift_examples_modify_imagestreams` modifies the image stereams's configuration.
2. It modifies the registry as pointed by `oreg_url`.

So, as [upstream's example](https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.ose.example#L94-L95) wrote, how about "If oreg_url points to a registry other than registry.access.redhat.com we can modify image streams to point at that registry"